### ISSUE TITLE
Building flutter apk failed with gradle errors

### ISSUE THREAD

**Issue by kcwu** [mlcommons/mobile_app_open#368]
I want to build flutter apk with master branch. However it failed.

I can run this command `make WITH_PIXEL=1` without issues.

When I run `make WITH_PIXEL=1 docker/flutter/android/apk`, it failed with following messages

```
💪 Building with sound null safety 💪

Exception in thread "main" java.io.FileNotFoundException: /image-workdir/.cache/.gradle/wrapper/dists/gradle-7.2-all/260hg96vuh6ex27h9vo47iv4d/gradle-7.2-all.zip.lck (No such file or directory)
        at java.base/java.io.RandomAccessFile.open0(Native Method)
        at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:345)
        at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:259)
        at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:214)
        at org.gradle.wrapper.ExclusiveFileAccessManager.access(ExclusiveFileAccessManager.java:49)
        at org.gradle.wrapper.Install.createDist(Install.java:48)
        at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:128)
        at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
Running Gradle task 'assembleRelease'...                           134ms
[!] Gradle threw an error while downloading artifacts from the network. Retrying to download...
Exception in thread "main" java.io.FileNotFoundException: /image-workdir/.cache/.gradle/wrapper/dists/gradle-7.2-all/260hg96vuh6ex27h9vo47iv4d/gradle-7.2-all.zip.lck (No such file or directory)
        at java.base/java.io.RandomAccessFile.open0(Native Method)
        at java.base/java.io.RandomAccessFile.open(RandomAccessFile.java:345)
        at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:259)
        at java.base/java.io.RandomAccessFile.<init>(RandomAccessFile.java:214)
        at org.gradle.wrapper.ExclusiveFileAccessManager.access(ExclusiveFileAccessManager.java:49)
        at org.gradle.wrapper.Install.createDist(Install.java:48)
        at org.gradle.wrapper.WrapperExecutor.execute(WrapperExecutor.java:128)
        at org.gradle.wrapper.GradleWrapperMain.main(GradleWrapperMain.java:61)
Running Gradle task 'assembleRelease'...                           115ms
[!] Gradle threw an error while downloading artifacts from the network. Retrying to download...
Gradle task assembleRelease failed with exit code 1
make: *** [flutter/android/android.mk:60: flutter/android/apk-only] Error 1
make: *** [flutter/android/android.mk:66: docker/flutter/android/apk] Error 2
```

**Comment by d-uzlov** [comment]
I wasn't able replicate this.
The command also failed for me but for a completely different reason. I don't see any issues related to gradle.

Could you try removing docker volume named `mlperf-mobile-flutter-cache-workdir-<uid>` to clear cache?

**Comment by anhappdev** [comment]
I can build the app with command `make WITH_PIXEL=1 docker/flutter/android/apk` successfully using the latest commit on `master` branch (`5c3c81dc`) on an Ubuntu VM.

@kcwu Can you provide some info about your build environment?

**Comment by ptt2panda** [comment]
The issue is fixed w/ the latest PR #371 and #370, I think we can close this. Thanks!
